docker/php/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM yiisoftware/yii-php:8.0-fpm AS php-build
```

Comment:
Was it added intentionally or is it part of your project?

Reply:
it is for local development this library

Reply:
should be removed

Reply:
removed what? Dockerfile?